installing/installing_bare_metal_ipi/ipi-install-overview.adoc

```diff
@@ -4,19 +4,24 @@
 include::_attributes/common-attributes.adoc[]
 :context: ipi-install
 
-Installer-provisioned installation provides support for installing {product-title} on bare metal nodes. This guide provides a methodology to achieving a successful installation.
-
-During installer-provisioned installation on bare metal, the installation program on the bare metal node labeled as `provisioner` creates a bootstrap virtual machine (VM). The role of the bootstrap VM is to assist in the process of deploying an {product-title} cluster. The bootstrap VM connects to the `baremetal` network and to the `provisioning` network, if present, via the network bridges.
+Installer-provisioned installation provides support for installing {product-title} on bare metal nodes. This guide provides a methodology to achieving a successful installer-provisioned base-metal installation. The following diagram illustrates the installation environment in phase one of deployment.
```

Comment:
On first read I found this a little confusing just because the terms installer / installation are used so much, and it's not instantly clear to me what "installer-provisioned" means.

Suggest maybe adding a line here that explains, something like "The installer-provisioned method of installation for {product-title} means that an installer program deploys and configures the infrastructure that the cluster runs on."

Reply:
Fair point. The first sentence is meant to do the job of explaining what "installer-provisioned installation" means. I'm updating it to be more precise - "Installer-provisioned installation on bare metal nodes deploys and configures the infrastructure that a {product-title} cluster runs on."